ViewLayer/ViewContainer.h

```diff
@@ -2,17 +2,21 @@
 
 #include <QScopedPointer>
 
-class I_DisplayDashboardUI;
+enum Mode { RACE, DISPLAY };
```

Comment:
Just one quick question about this part. Are you looking for me to make a namespace in this header file and encapsulate the enum there or encapsulate the enum within the ViewContainer class?